data/adapters/AFGR_caqtl_adapter.py

```diff
@@ -0,0 +1,79 @@
+import csv
+import gzip
+from math import log10
+
+from adapters import Adapter
+from adapters.helpers import build_variant_id, build_regulatory_region_id
+
+# Example row from sorted.dist.hwe.af.AFR.caQTL.genPC.maf05.90.qn.idr.txt.gz
+# chr	snp_pos	snp_pos2	ref	alt	variant	effect_af_eqtl	p_hwe	feature	dist_start	dist_end	pvalue	beta	se
+# 1	66435	66435	ATT	A	1_66435_ATT_A	0.125	0.644802	1:1001657:1002109	-935222	-935674	0.616173	0.055905	0.111128
+
+
+class AFGRCAQtl(Adapter):
+    ALLOWED_LABELS = ['regulatory_region', 'AFGR_caqtl']
+
+    SOURCE = 'AFGR'
+    SOURCE_URL = 'https://github.com/smontgomlab/AFGR'
+
+    CLASS_NAME = 'accessible_dna_element'
+    ONTOLOGY_TERM = 'EFO_0005292'  # lymphoblastoid cell line
+
+    def __init__(self, filepath, label):
+        if label not in AFGRCAQtl.ALLOWED_LABELS:
+            raise ValueError('Ivalid label. Allowed values: ' +
+                             ','.join(AFGRCAQtl.ALLOWED_LABELS))
+
+        self.filepath = filepath
+        self.label = label
+
+        super().__init__()
+
+    def process_file(self):
+        with gzip.open(self.filepath, 'rt') as qtl_file:
+            qtl_csv = csv.reader(qtl_file, delimiter='\t')
+            next(qtl_csv)
+
+            for row in qtl_csv:
+                region_chr, region_pos_start, region_pos_end = row[8].split(
+                    ':')
+                regulatory_region_id = build_regulatory_region_id(
+                    region_chr, region_pos_start, region_pos_end, class_name=AFGRCAQtl.CLASS_NAME
+                )
+
+                if self.label == 'regulatory_region':
+                    _id = regulatory_region_id
+                    _props = {
+                        'chr': region_chr,
```

Comment:
we need to add a `name` property to all nodes now... in this case, would `'name': _id` work?

Reply:
yes it works